libs/typings/src/ui/extensions.ts

```diff
@@ -16,6 +22,33 @@ export enum ExtensionStatus {
   Published = 'Published',
 }
 
+/**
+ * Type defining extension image types
+ **/
+export type ExtensionImageType = 'logo-image' | 'cover-image';
+
+export type Extension = {
```

Comment:
Can't we reuse the AkashaAppInterface here? no need to redefine all the fields, just add the extra ones